src/index.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+
+import tracer from 'opentracing'
+import Backend from './tracer'
+
+tracer.configure = (options) => {
+    tracer.initGlobalTracer(new Backend(options))
```

Comment:
src/tracer.js is the `Backend`, and the name coming from https://github.com/opentracing/opentracing-javascript/blob/master/README.md